src/api/spec/lib/rubocop/cop/view_component/avoid_global_state_spec.rb

```diff
@@ -2,9 +2,13 @@
 require './lib/rubocop/cop/view_component/avoid_global_state'
 
 RSpec.describe RuboCop::Cop::ViewComponent::AvoidGlobalState, :config do
+  let(:config) do
+    RuboCop::Config.new({ 'ViewComponent' => { 'Include' => ['app/components/**/*.rb'] } }, '/some/.rubocop.yml')
```

Comment:
We are now mocking the RuboCop config used in this spec file. It proves that the changes are working.